velox/docs/monitoring/stats.rst

```diff
@@ -39,3 +39,25 @@ Memory Arbitration
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+
+HashBuild, HashAggregation
+------------------
```

Comment:
--- needs to be extended

Please, generate the docs locally and verify there are not errors / warnings and the result looks nice.

Reply:
Fixed

Reply:
The Clion has a plugin to preview rst files. I checked it after modifying it.